managers.py

```diff
@@ -0,0 +1,26 @@
+import sqlite3
+from models import Actor
+
+
+class ActorManager:
+    def __init__(self, db_name: str):
+        self.connection = sqlite3.connect(db_name)
+        self.cursor = self.connection.cursor()
+
+    def create(self, first_name: str, last_name: str):
+        self.cursor.execute("INSERT INTO actors (first_name, last_name) VALUES (?, ?)", (first_name, last_name))
+        self.connection.commit()
+        return self.cursor.lastrowid
+
+    def all(self):
+        self.cursor.execute("SELECT id, first_name, last_name FROM actors")

```

Comment:
It's better to follow SQL style guidelines even while writing raw SQL in Python. Therefore I'd suggest to start FROM clause from new line.